### ISSUE TITLE
Stop-loss

### ISSUE THREAD

**Issue by sbralin** [freqtrade/freqtrade#3621]
Hi in strategy file i have stoploss = -020 if stoploss is triggered how to set self.lock_pair(pair, until). until for end of day.

**Comment by xmatthias** [comment]
This will currently only work out of the box if you're on daily candles - where locking until the end of candle is automatic (and non-optional).


There is no callbback to inform the strategy about the sell reason - but i guess you could use the [Trade interface](https://www.freqtrade.io/en/latest/strategy-customization/#additional-data-trades) to get the trade and determine if it fits your criteria - and then [manually block](https://www.freqtrade.io/en/latest/strategy-customization/#prevent-trades-from-happening-for-a-specific-pair) the pair for the desired timeframe.

The above mentioned method will not work in backtesting, and you must make sure to not run it during backtesting (basically doing all of this within `if self.config['runmode'].value in ('live', 'dry_run'):`).
